### ISSUE TITLE
transcript analysis

### ISSUE THREAD

**Issue by mcfrank** [langcog/xsface#4]
once transcripts are in plaintext, we should compute MLU on these so that it can be added to the descriptives. are there any other interesting analyses to add? @dyurovsky: any ideas?

**Comment by dyurovsky** [comment]
Well we could do some other straightforward measures like sentence position
of the referring noun, etc. Could also try to do some of the kinds of
analyses that you and Hannah Rohde did with discourse topic.

Dan

**Comment by mcfrank** [comment]
True! Do you think there is a good way to link these to the headcam data? 

Mike

**Comment by dyurovsky** [comment]
Well, in principle, we could see whether kids look up to the face early or
late in a discourse topic change. Dunno how sparse the data would be, but
maybe?

Dan
